README.md

```diff
@@ -281,3 +281,17 @@ purposes, since Pebble and its generated keys are not audited or held to the
 same standards as the Let's Encrypt production CA and their keys. Moreover
 these keys are exposed by Pebble and will be lost as soon as the process
 terminates: so they are not safe to use for anything other than testing.**
+
+### OCSP Responder URL
+
+Pebble does not support the OCSP protocol as a responder and so does not set
+the OCSP Responder URL in the issued certificates. However, if you setup a
+proper OCSP Responder run side by side with Pebble, you may want to set this URL.
+This is possible by setting the environment variable `PEBBLE_CA_OCSP_RESPONDER_URL`:
+if this variable is set to a non-empty string, its value will be used in the
+appropriate field of all issued certificates.
+
+For instance, to instruct a client to check the URL `http://127.0.0.1:4002` and
```

Comment:
nit: I think changing "and" to "to" makes this a little bit more readable, and I'd maybe make it a little bit clearer how the client learns the OCSP server URL (via a field in the certificate, not via ACME).

```suggestion
For instance, to have Pebble issue certificates that instruct a client to check the URL `http://127.0.0.1:4002` to
```